eng/versioning/external_dependencies.txt

```diff
@@ -50,6 +53,36 @@ org.powermock:powermock-api-mockito2;2.0.2
 org.powermock:powermock-module-junit4;2.0.2
 org.slf4j:slf4j-api;1.7.28
 org.slf4j:slf4j-simple;1.7.25
+org.springframework:spring-web;5.2.0.RELEASE
+org.springframework:spring-jdbc;5.2.0.RELEASE
+org.springframework:spring-jms;5.2.0.RELEASE
+org.springframework.boot:spring-boot-autoconfigure;2.2.0.RELEASE
+org.springframework.boot:spring-boot-actuator-autoconfigure;2.2.0.RELEASE
+org.springframework.boot:spring-boot-autoconfigure-processor;2.2.0.RELEASE
+org.springframework.boot:spring-boot-configuration-processor;2.2.0.RELEASE
+org.springframework.boot:spring-boot-dependencies;2.2.0.RELEASE
+org.springframework.boot:spring-boot-starter;2.2.0.RELEASE
+org.springframework.boot:spring-boot-starter-test;2.2.0.RELEASE
+org.springframework.boot:spring-boot-starter-validation;2.2.0.RELEASE
+org.springframework.boot:spring-boot-starter-web;2.2.0.RELEASE
+org.springframework.security:spring-security-core;5.2.0.RELEASE
+org.springframework.security:spring-security-config;5.2.0.RELEASE
+org.springframework.security:spring-security-web;5.2.0.RELEASE
+org.springframework.security:spring-security-oauth2-client;5.2.0.RELEASE
+org.springframework.security:spring-security-oauth2-core;5.2.0.RELEASE
+org.springframework.security:spring-security-oauth2-jose;5.2.0.RELEASE
+com.microsoft.azure:spring-data-cosmosdb;2.2.2
```

Comment:
spring-data-cosmsodb current released version is v2.2.3. Please update this.

Reply:
Updated.